src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -212,6 +212,12 @@ def push_messages(self, messages: Iterable[Message]) -> Iterable[str]:
                 messages_list=message_proto_list, run_id=cast(Run, self._run).run_id
             )
         )
+        if len([id for id in list(res.message_ids) if id]) != len(list(messages)):
+            log(
+                WARNING,
+                "Not all messages could be pushed to the SuperLink. "
+                "This could be due to a malformed message.",
```

Comment:
I think it's a good idea showing this warning. Worth adding an extra sentence indicating that the returned list has `None` for those messages that weren't pushed successfully ?. the order is kept as passed to `push_messages`.